pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2191,6 +2191,30 @@ var _ = Describe("Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(domStats).ToNot(BeNil())
 		})
+		Context("on failed GetAllDomainStats", func() {
+			It("should return err if domainStats are nil", func() {
+				const (
+					domainStats = libvirt.DOMAIN_STATS_BALLOON |
+						libvirt.DOMAIN_STATS_CPU_TOTAL |
+						libvirt.DOMAIN_STATS_VCPU |
+						libvirt.DOMAIN_STATS_INTERFACE |
+						libvirt.DOMAIN_STATS_BLOCK |
+						libvirt.DOMAIN_STATS_DIRTYRATE
+					flags = libvirt.CONNECT_GET_ALL_DOMAINS_STATS_RUNNING | libvirt.CONNECT_GET_ALL_DOMAINS_STATS_PAUSED
+				)
```

Comment:
is this necessary?

Reply:
yes, otherwise the generated mock libvirt fails